packages/datadog_flutter_plugin/lib/src/datadog_configuration.dart

```diff
@@ -254,104 +254,119 @@ class DatadogConfiguration {
-//         }
-//       }
-//     }
-//   }
-
-//   void addPlugin(DatadogPluginConfiguration pluginConfiguration) =>
-//       additionalPlugins.add(pluginConfiguration);
-// }
+class DatadogAttachConfiguration {
+  /// Enable or disable detection of "long tasks"
+  ///
+  /// Long task detection attempts to detect when an application is doing too
+  /// much work on the main isolate which could prevent your app from rendering
+  /// at a smooth framerate.
+  ///
+  /// This option does not have any affect on options already set in the Native
+  /// SDK, and only initializes long task detection on the main Dart isolate.
+  ///
+  /// This option is ignored if RUM is not enabled in the Native SDK.
+  ///
+  /// Defaults to true.
+  bool detectLongTasks;
+
+  // The amount of elapsed time that is considered to be a "long task", in /
+  //seconds.
+  ///
+  /// If the main isolate takes more than [longTaskThreshold] seconds to process
+  /// a microtask, it will appear as a Long Task in Datadog RUM Explorer. This
+  /// has a minimum of 0.02 seconds.
+  ///
+  /// This threshold only applies to the Dart long task detector. The Native
+  /// SDKs / will retain their own thresholds.
+  ///
+  /// This option is ignored if RUM is not enabled in the Native SDK.
+  ///
+  /// Defaults to 0.1 seconds
+  final double longTaskThreshold;
+
+  /// Whether to report Flutter specific performance metrics (build and raster
+  /// times)
+  ///
+  /// This uses the [SchedulerBinding.addTimingsCallback] method to report build
+  /// and raster times for views, and has a documented negligible impact on
+  /// performance.
+  ///
+  /// Defaults to false
+  bool reportFlutterPerformance = false;
+
+  /// A list of first party hosts, used in conjunction with Datadog network
+  /// tracking packages like `datadog_tracking_http_client`
+  ///
+  /// This property only affects network requests made from Flutter, and is not
+  /// shared with or populated from existing the SDK.
+  ///
+  /// Overwriting this property will overwrite
+  /// [DatadogAttachConfiguration.firstPartyHostsWithTracingHeaders]
+  ///
+  /// See [DatadogConfiguration.firstPartyHosts] for more information
+  List<String> get firstPartyHosts =>
+      firstPartyHostsWithTracingHeaders.keys.toList();
+  set firstPartyHosts(List<String> hosts) {
+    firstPartyHostsWithTracingHeaders.clear();
+    for (var entry in hosts) {
+      firstPartyHostsWithTracingHeaders[entry] = {TracingHeaderType.datadog};
+    }
+  }
+
+  /// A list of first party hosts and the types of tracing headers Datadog
+  /// should automatically inject on resource calls. This is used in conjunction
+  /// with Datadog network tracking packages like `datadog_tracking_http_client`
+  ///
+  /// This property only affects network requests made from Flutter, and is not
+  /// shared with or populated from existing the SDK.
+  ///
+  /// See [DdSdkConfiguration.firstPartyHostsWithTracingHeaders] for more information
```

Comment:
probably a wrong reference?

```suggestion
  /// See [DatadogConfiguration.firstPartyHostsWithTracingHeaders] for more information
```

Reply:
Thanks, I've made task to do a full scan over documentation for renames.